src/label_view.js

```diff
@@ -1,229 +1,171 @@
+import $ from "jquery";
+
 console.log = function() {}; // disable logging
 
-var preview_id; // globally stores which 'preview' button was last clicked
-var preview_open = false; // globally stores if preveiw window is open
-
-var storageCache = [];
-
-// load storage backed
-var storageLoadedPromise = new Promise(function(resolve, reject) {
-  chrome.storage.sync.get("previews", function(result) {
-    try {
-      // load storage
-      if (isEmpty(result)) {
-        console.log("storage empty, storing new");
-        chrome.storage.sync.set({ previews: storageCache });
-      } else {
-        console.log("storage exists, storing to variable 'storageCache'");
-        storageCache = result["previews"];
-      }
+let previewId; // globally stores which 'preview' button was last clicked
+let previewOpen = false; // globally stores if preveiw window is open
```

Comment:
These globals might be good candidates for storing on `pluginState` in the future 👍 

I'm not 100% sure about using `let` block-level scoping in the global JS VM, but I have a feeling that these vars will truly be global and thus vulnerable to external manipulation or naming collisions.